### DISCUSSION TITLE
Vercel build - Different region?

### DISCUSSION THREAD

**Discussion by tobidemski** [vercel/next.js#50282] [Help]
### Summary

Hello,

I have seen that if I trigger a new deployment under "Deployment status" / "Building" step in the first line I see the following
`Running build in Washington, D.C., USA (East) – iad1`

I would like to run this in the EU (fra1) and I can not find anything if it is possible to change this.
I already have changed the serverless function location under the project settings / function to "Frankfurt, Germany (West) – fra1"

Would be nice if someone can help me out here :)

**Comment by Noah3141** [comment]
Apparently, apart from the Function setting, there is also a vercel.json "regions" key you can use....
Neither has worked for me though...

**Comment by l3wi** [comment]
Also experiencing an issue with this

**Comment by jiwoniverse** [comment]
Same with me..😭 I also changed the Function Region to `Seoul, South Korea (Northeast) - icn1`, but Running build is starting in `Washington, D.C., USA (East) – iad1`.

**Comment by maxlibin** [comment]
same problem

**Comment by kognito** [comment]
Same issue here. Any update on this please?

**Comment by ornakash** [comment]
Same here.

**Comment by taylor-lindores-reeves** [comment]
+1 is this only for preview builds?

**Comment by ivanbogdan-git** [comment]
+1

**Comment by NicoToff** [comment]
The build seems to always run in Washington, but the build is then deployed to Vercel's CDN. You can check where a request comes from by running `curl -I <your_website>` and check the first part of the **x-vercel-id** header.

For me, it is generally `cdg1::....` and that means I got answered from France (I'm in Belgium, so fair enough).

**Comment by erik-metz** [comment]
+1

**Comment by freaghill** [comment]
This is what I got:

2. Ensure Builds Run in the Desired Region
Currently, Vercel builds are executed in the default build region, which is Washington, D.C., USA (iad1). Unfortunately, the build region cannot be changed to Germany or another EU region at this time. Builds will always occur in the default build region.

**Comment by bonesoul** [comment]
still the same?
